diff --git a/WineskinEngineBuild b/WineskinEngineBuild
index 9b2dbf1..a445414 100755
--- a/WineskinEngineBuild
+++ b/WineskinEngineBuild
@@ -102,6 +102,7 @@ fi
 ######## Start building the Wine source
 # set build environment variables so it builds properly.
 export PATH="/tmp/WineskinGCC/bin:$BEPATHBINS:$PATH"
++export PKG_CONFIG_PATH='${BEPATHLIBS}/pkgconfig'
 #export CC="gcc"
 #export CXX="g++"
 export CC="gcc-4.2"
@@ -170,7 +171,7 @@ then
 			sudo rm "/usr/bin/gcc-4.2"
 			CleanUp;exit
 		fi
-	done	
+	done
 	#delete left over gcc folder
 	cd ..
 	rm -rf gcc
@@ -227,6 +228,11 @@ then
 	then
 		export MINMACOSXVERSIONTOBUILDFOR="10.9"
 		export OSXSDK="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk"
+    #test for 10.11 SDKs
+    elif [ -d "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk" ]
+	then
+		export MINMACOSXVERSIONTOBUILDFOR="10.11"
+		export OSXSDK="/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk"
 	else
 		echo "";echo ""
 		echo "ERROR!!! No usable SDKs found on your system... this may not work right!  Please Install Apple Xcode to get SDKs"
